Common/Transforms/itkBSplineInterpolationDerivativeWeightFunction.hxx

```diff
@@ -91,15 +91,15 @@ BSplineInterpolationDerivativeWeightFunction<TCoordRep, VSpaceDimension, VSpline
 
     if (i != this->m_DerivativeDirection)
     {
-      for (unsigned int k = 0; k < this->m_SupportSize[i]; ++k)
+      for (unsigned int k = 0; k <= VSplineOrder; ++k)
```

Comment:
Suggestion by Marius (@mstaring): do `k < (VSplineOrder + 1)`, instead of `k <= VSplineOrder`, for the sake of clarity. (In each of those `for` loops over `k`.)

Reply:
Addressed by [this force-push](https://github.com/SuperElastix/elastix/compare/019070bbf1f85ea8dbe322cf7bb873b502fa672e..ec639f4505e4b43ca97aeda9f50710d0d131ee79)